apps/OboeTester/app/src/main/java/com/mobileer/oboetester/EchoActivity.java

```diff
@@ -204,8 +204,4 @@ public void onStopEcho(View view) {
     boolean isOutput() {
         return false;
     }
-
-    @Override
-    public void setupEffects(int sessionId) {
```

Comment:
If an empty method was declared in TestAudioActivity, instead of an abstract method, then all these empty Overrides would not be needed.

Reply:
We are moving settingEffects from the job of an individual test into each stream config. This way output tests can get output effects, input tests can get input effects, and tests that test both can do both.